style.css

```diff
@@ -381,6 +385,120 @@ hr {
   box-shadow: none;
 }
 
+#popup-project {
```

Comment:
- [ ] The Technologies section is not present in your design, kindly implement this section
- [ ] the close button is not the same as the Figma design, please fix it so it can be as close as possible to the original design

- Your design:
![Screenshot_20220511_063144](https://user-images.githubusercontent.com/25227022/167775710-52c79c39-173b-4b82-8428-ce1f45cb3b25.png)


- Figma design:
![Screenshot_20220511_062322](https://user-images.githubusercontent.com/25227022/167775538-b64bc4ee-3b95-4a99-94ec-d9092e080283.png)